src/textarea.test.jsx

```diff
@@ -101,6 +104,24 @@ describe('Textarea', function() {
 		expect(changeSpy).toHaveBeenCalled();
 		expect(stateSpy).toHaveBeenCalledWith({ value: newValue });
 	});
+	it('should call onChange `props` function when input is changed', function() {
+		const newValue = `${VALUE}r`;
+
+		const boundComponent = TestUtils.renderIntoDocument(
+			<Textarea
+				name={NAME_ATTR}
+				label={LABEL_TEXT}
+				value={VALUE}
+				id={ID}
+				onChange={onChange}
+			/>
+		);
+
+		textareaEl = TestUtils.findRenderedDOMComponentWithTag(boundComponent, 'textarea');
+		TestUtils.Simulate.change(textareaEl, { target: { value: newValue } });
+
+		expect(onChange).toHaveBeenCalled();
```

Comment:
When using a simulated user action, it's usually good to check that the spy was _not_ called before the interaction in addition to checking that is _is_ called afterward. Otherwise, code that called the spy on initialization and then ignored the user input would still pass the unit test despite being wrong.